services/core-api/app/api/mines/permits/permit_conditions/models/permit_condition_category.py

```diff
@@ -24,4 +24,8 @@ def __repr__(self):
 
     @classmethod
     def get_all(cls):
-        return cls.query.order_by(cls.display_order).all()
\ No newline at end of file
+        return cls.query.order_by(cls.display_order).all()
+
+    @classmethod
+    def find_by_permit_condition_category_code(cls, code):
+        return cls.query.filter_by(condition_category_code=code).one_or_none()
```

Comment:
both of these should probably have an active_ind check on them.

Reply:
maybe not for the get_all, but I'll definitely add one for find_by_permit_condition_category_code